mobile/src/test/java/ch/epfl/sdp/mobile/application/chess/parser/StringCombinatorsTest.kt

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.sdp.mobile.application.chess.parser
+
+import ch.epfl.sdp.mobile.application.chess.parser.StringCombinators.string
+import org.junit.Assert.*
+import org.junit.Test
+
+class StringCombinatorsTest {
+
+  @Test
+  fun given_aString_when_parsed_then_returnSetOfResult() {
+    val res = string().parse("This is a test")
+    val expected = Parser.Result("is a test", "This")
+    assertEquals(expected, res.first())
```

Comment:
With Truth, `assertEquals` becomes `assertThat(...).isEqualTo(...)`

Reply:
Thx for this example ! It's easier to understand